climateconnect/climateconnect
# Feat: Label indicating a reply must be sent in chatbox

## Description

Closes #1150 

This PR implements a label onto chat previews for messages in which the logged in user was not the last one to write a message. Should a user not want to see the labels, they can use a slider at the top of the inbox to turn them off. A cookie will track this state for 30 days.

The label shows under the condition that the user was not the last one to send a message, or the chat has not had any messages sent in it yet.

The new label is only implemented for non mobile screens.

Interface before:
![Screenshot from 2022-11-21 14-52-57](https://user-images.githubusercontent.com/49129464/203072244-2893912a-f16c-4cb5-a61a-18848426cf6e.png)


Screenshot displays new interface.

![Screenshot from 2022-11-21 14-33-38](https://user-images.githubusercontent.com/49129464/203068478-05529b6d-f790-4593-bb4a-0f6ed1d0a450.png)


## Test plan

1. Go to inbox
2. Send a message to yourself and see the label appear

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
